frontend/src/components/@common/Map/OverlayMarker.tsx

```diff
@@ -37,6 +39,11 @@ function OverlayMarker({ celeb, restaurant, map, quadrant, isRestaurantHovered }
         <StyledMarkerContainer ref={ref}>
           <StyledMarker onClick={clickMarker} isClicked={isClicked} isRestaurantHovered={isRestaurantHovered}>
             <ProfileImage name={celeb.name} imageUrl={celeb.profileImageUrl} size="100%" />
+            {restaurant.isLiked && (
+              <LikeButton aria-label="좋아요" type="button">
+                <Love fill={restaurant.isLiked ? 'red' : '#000'} fillOpacity={0.5} aria-hidden="true" />
```

Comment:
여기에서는 fill에 대해 분기 처리가 없어도 될 것 같아요~!
```suggestion
                <Love fill='red' fillOpacity={0.5} aria-hidden="true" />
```